sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/ConfiguringLoggersTest.cs

```diff
@@ -31,6 +31,8 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class ConfiguringLoggersTest
     {
+        private const string DOT_NET_CORE_VERSION = "2.2.0";
```

Comment:
I would prefer having the version of .NET Core declared only at one location in `Constants`